.github/workflows/uninstall_install_sandbox.yaml

```
@@ -28,15 +28,28 @@ jobs:
           namespace: ${{ secrets.OPENSHIFT_NAMESPACE }}
       
       - name: Uninstall Traction via Helm
-        continue-on-error: true
+        continue-on-error: false
         run: |
           helm uninstall traction-sandbox -n ${{ secrets.OPENSHIFT_NAMESPACE }} --wait --timeout=20m
 
       - name: Remove Traction Openshift Objects
-        continue-on-error: true 
+        continue-on-error: false 
         run: |
           oc delete -n ${{ secrets.OPENSHIFT_NAMESPACE }} all,secret,pod,networkpolicy,configmap,pvc --selector "app.kubernetes.io/instance"=traction-sandbox
 
+      - name: Rocket.Chat Notification
+        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@master
```

```suggestion
        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@1.1.1
```
It's best practice to always target a released version of an action, that way you are not surprised by unexpected changes as you would be when targeting the `master` branch.  We covered this during your previous work with this action here; https://github.com/bcgov/von-bc-registries-audit/blob/main/.github/workflows/actions/deploy/action.yml#L83-L93